transformers/src/unit-test/java/uk/gov/moj/sdt/transformers/UpdateItemTransformerTest.java

```diff
@@ -67,6 +67,10 @@ class UpdateItemTransformerTest extends AbstractSdtUnitTestBase {
      */
     private UpdateItemTransformer updateItemTransformer;
 
+    private static final String FOUND_CORRECT_REQUEST_STATUS = "Found correct request status";
+    private static final String FOUND_CORRECT_SDT_REQUEST_ID = "Found correct sdt request id";
+    private static final String MCOL_12200202_121212 = "MCOL-12200202-121212";

```

Comment:
The constant name isn't very descriptive!  The value is an SDT request id so the constant should be named as such.

Reply:
renamed to SDT_REQUEST_ID